docs/policy-only.md

```diff
@@ -0,0 +1,194 @@
+# Running Antrea In Policy Only Mode
+
+Antrea supports chaining with other CNI implementations such as EKS. In this mode, 
+Antrea enforces Kubernetes NetworkPolicy, and delegates Pod IP management
+and network connectivity to the primary CNI.
+
+The policy-only mode is compatible currently with routed CNI implementations. 
+
+## Deploy and Test Antrea In Cloud 
+Assuming you already have a cloud managed Kubernetes cluster, and have KUBECONFIG environment
+variable set to configuration file of that cluster.
+
+Create an Antrea manifest for policy-only mode, for instance
+```bash
+./hack/generate-manifest.sh --encap-mode networkPolicyOnly > ~/my-yaml/antrea-policy.yaml
```

Comment:
we can do that, but user would still need to modify the checked in manifest, because in each CNI environment, service CIDR, MTU may all be different, So it sort defeats the reasoning to have a check-in version of manifest in the first place.